docs/src/ScrollableContainer.doc.js

```diff
@@ -312,12 +320,8 @@ function ScrollableContainerExample() {
       </Flex>
     </ScrollableContainer>
 )}`}
-      />
-    </MainSection.Subsection>
-    <MainSection.Subsection
-      title="Modal, DropDown, and Typeahead with ScrollableContainer"
-      description="In this example, the content in Modal extends the modal's dimensions. It requires ScrollableContainer so that Typeahead and Dropdown components stay correctly anchored after scrolling."
-    >
+    />
+    <MainSection.Subsection title="TEMPORARY FOR TESTING PURPOSES">
```

Comment:
I think you can go ahead and give this the title it will have once we ship the experiment, sort of an optimistic update 🙂

Also: it'd be nice to easily demonstrate how this works on both Modal and Sheet without the reader needing to modify the example code. Perhaps a radio button to switch between the two? If the props remain the same (which it looks like they do from your live coding in the video), you could do something like `const ParentComponent = condition ? Modal : Sheet` to avoid duplicating all the markup in `children`

Reply:
DONE.